src/main/java/ua/mateacademy/tania/lesson17/jdbc/Office.java

```diff
@@ -85,4 +86,20 @@ public String toString() {
                 ", sale=" + sale +
                 '}';
     }
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) return true;
+        if (o == null || getClass() != o.getClass()) return false;
+        Office office = (Office) o;
+        return Objects.equals(id, office.id) &&
+                Objects.equals(city, office.city) &&
+                Objects.equals(region, office.region);
+    }
+
+    @Override
+    public int hashCode() {
+
+        return Objects.hash(id, city, region);
+    }
```

Comment:
You can not do these methods, they are not needed here

Reply:
> You can not do these methods, they are not needed here

I need it for testing